cmd/tools/grafana/dashboard_test.go

```diff
@@ -487,6 +487,24 @@ func TestIDIsBlank(t *testing.T) {
 		})
 }
 
+func TestExemplarTrue(t *testing.T) {
+	visitDashboards(
+		[]string{"../../../grafana/dashboards/cmode", "../../../grafana/dashboards/storagegrid"},
+		func(path string, data []byte) {
+			checkExemplarTrue(t, path, data)
+		})
+}
+
+func checkExemplarTrue(t *testing.T, path string, data []byte) {
+
+	gjson.ForEachLine(string(data), func(line gjson.Result) bool {
```

Comment:
probably better to use a different API since grafana dashboards are not json lines